docs/symptom-survey/contingency-tables.md

```diff
@@ -90,34 +90,131 @@ alphabetically. Each aggregate reports four columns (unrounded):
+| `pct_hesitant_dontneed_reason_dont_spend_time` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine because they don't spend time with high-risk people, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
+| `pct_hesitant_dontneed_reason_not_high_risk` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine because they are not in a high-risk group, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
+| `pct_hesitant_dontneed_reason_precautions` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine because they will use other precautions, such as a mask, instead, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
+| `pct_hesitant_dontneed_reason_not_serious` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine because they don't believe COVID-19 is a serious illness, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
+| `pct_hesitant_dontneed_reason_not_beneficial` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine because they don't think vaccines are beneficial, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
+| `pct_hesitant_dontneed_reason_other` | Estimated percentage of respondents who say they don't need to get a COVID-19 vaccine for another reason, among respondents who answered No, probably not" or "No, definitely not" when asked if they would get vaccinated if offered (item V3) and provided at least one reason for why they believe a COVID-19 vaccine is unnecessary. <br/> **Earliest date available:** 2021-02-08 | V6 |
 
 Note: CSVs for the month of January 2021 only use data from January 6-31 due to
```

Comment:
I think this note should get moved up to the top now, since it's buried under a huge list of indicators.